readme file translations en, es (#7)
finloop/airflow-postgres-superset-on-docker: README.en.md

# Airflow + Postgres + Superset

## Architecture

![](docs/Architecture.png)

## Run

    git clone git@github.com:finloop/airflow-postgres-superset-on-docker.git
    cd airflow-postgres-superset-on-docker

To run the services you will need the id of the current user, which should be placed

> in a file `.env` and assign to a variable `AIRFLOW_UID`. On linux id of the current user
> can be downloaded by the command:

Reviewer: ```suggestion
in a file `.env` and assign to a variable `AIRFLOW_UID`. On linux, issue following command to get users id:
```

```sh
id -u
```

Sample file `.env` looks like this:

```text
AIRFLOW_UID=1000
```

And finally, we launch all the services:

```sh
docker-compose up
```

After running them, you should put the sample data in the database. We do this:

```sh
python init-postgres.py
```

> ### WebsitesW

Reviewer: ```suggestion
### Web services
```
